client/src/panel-toc-editor.ts

```diff
@@ -176,8 +156,8 @@ export const refreshPanel = async (panel: vscode.WebviewPanel, client: LanguageC
   const trees = await requestBundleTrees(client, { workspaceUri: uri.toString() })
   const allModules = await requestBundleModules(client, { workspaceUri: uri.toString() })
   const orphanModules = await requestBundleOrphanedModules(client, { workspaceUri: uri.toString() })
+  /* istanbul ignore if */
```

Comment:
Could this just be `expect(trees, 'Server cannot properly find workspace')`?

Reply:
Sure. Less coverage ignoring!